crates/jose/Cargo.toml

```diff
@@ -9,6 +9,7 @@ license = "Apache-2.0"
 anyhow = "1.0.64"
 base64ct = { version = "1.5.2", features = ["std"] }
 chrono = { version = "0.4.22", features = ["serde"] }
+data-encoding = "2.3.2"
```

Comment:
I started using `base64ct` for base64 encoding instead of `data-encoding`, since it's used by RustCrypto crates, like `password-hash` and `pem-rfc7468`. I'll do a PR later to remove `data-encoding` completely, but if here you could already use `base64ct` instead that would be great